lib/rubocop/cop/rspec/factory_bot/factory_class_name.rb

```diff
@@ -6,6 +6,11 @@ module RSpec
       module FactoryBot
         # Use string value when setting the class attribute explicitly.
         #
+        # Although FactoryBot allows passing a class, passed application models
+        # are loaded before the application itself gets loaded.
+        # This affects libraries like "simplecov" in that they fail to capture
```

Comment:
Is there a difference if you require `simplecov` before or after `factory_bot`?

Reply:
Honestly, I don't know because I never had this situation.

This description is originated from #839, but if it may be unclear, I can accept any better suggestions.

Reply:
I'd go with something like this https://github.com/rubocop-hq/rubocop-rspec/pull/839#issuecomment-554687768, and would keep simplecov note, however, those two hypotheses need to be confirmed first.

Reply:
You're right. But I don't have resources to confirm these hypotheses and the overhead can be ignored in small applications.

After all, does this cop warn a stylistic issue?